package org.acme.domain.exception

class UserNotFoundException: RuntimeException("Usuário não encontrado")